pkgs/applications/editors/vscode/extensions/default.nix

```diff
@@ -3521,8 +3521,8 @@ let
         mktplcRef = {
           name = "uiua-vscode";
           publisher = "uiua-lang";
-          version = "0.0.19";
-          sha256 = "sha256-Tww1urq6CfLma254Sn5lwOYwbvxAeDZuBuFBQlzks1c=";
+          version = "0.0.21";
+          sha256 = "sha256-u57U/MmxvionFZp/tLK/KpddaxA/SUffeggKBSzmsXo=";
```

Comment:
(non-blocking)
```suggestion
          hash = "sha256-u57U/MmxvionFZp/tLK/KpddaxA/SUffeggKBSzmsXo=";
```

Reply:
IIRC hash is not supported in vscode extensions. (not 100%, but nobody seems to be using hash there)

Reply:
that seems odd, should work for any sri hash, can you try and see if it breaks anything?

Reply:
It doen't work